elastic/eui
# [EuiSuperDatePicker] Convert quick select styles to Emotion

## Summary

Part 3 of 3 (see #7904, #7908)

Converts EuiSuperDatePicker's quick select popover/button styles to Emotion. I strongly recommend [following along by commit](https://github.com/elastic/eui/pull/7909/commits) as I ended up creating a new internal `EuiQuickSelectPanel` subcomponent for styling, and also noticing a bunch of missing i18n in the refresh interval component. I also strongly recommend [hiding whitespace changes](https://github.com/elastic/eui/pull/7909/files?w=1) while viewing diffs as there are several sets of changes that are just indentation.

## QA

- Go to https://eui.elastic.co/pr_7909/#/templates/super-date-picker#quickselect-custom-rendering
  - [ ] Confirm that borders and spacing between panels/sections look correct regardless of # of panels permutations (toggle various switches)
- Go to https://eui.elastic.co/pr_7909/#/templates/super-date-picker#quick-select-panels
  - Click the quick select prepend button
  - [ ] Confirm the popover looks the same production
  - Click the quick select right/left arrows under 5+ recently used time range entries have been recorded
  - [x] Confirm the scrolling behavior works as expected
- Go to https://eui.elastic.co/pr_7909/#/templates/super-date-picker#auto-refresh
  - [ ] Click the quick select button and confirm the popover looks non-broken/same as production

### General checklist

- Browser QA
- Docs site QA - N/A
- Code quality checklist
    ~- [ ] Added or updated **[jest](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/unit-testing.md) and [cypress](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/cypress-testing.md) tests**~
- Release checklist
    ~- [ ] If applicable, added the **breaking change** issue label (and filled out the breaking change checklist)~
- Designer checklist - N/A
